static/app/views/preprod/components/visualizations/appSizeCategories.tsx

```diff
@@ -21,6 +21,17 @@ export function AppSizeCategories(props: AppSizeCategoriesProps) {
     categorySizes[categoryKey] = totalSize;
   });
 
+  const categorizedTotal = Object.values(categorySizes).reduce(
+    (sum, size) => sum + size,
+    0
+  );
+
+  // Binary overhead not in category_breakdown (padding, alignment)
+  const unmappedSize = treemapData.root.size - categorizedTotal;
```

Comment:
I don't think this is the right fix, it's basically just adding a fake value to make the numbers match up. And this definition doesn't necessary match up with our definition of UNMAPPED so I don't think that's accurate to do.

What nodes are missing a category? Can't we fix that?

Reply:
ah, i misunderstood what was missing here